tests/test/util/test_config.cpp

```diff
@@ -22,6 +22,7 @@ TEST_CASE("Test default system config initialisation", "[util]")
 
```

Comment:
Missing a test to check that the flag actually switches off topology in the scheduler.